### DISCUSSION TITLE
Τελική παρουσίαση με μπόνους

### DISCUSSION THREAD

**Discussion by Angeloth1** [courses-ionio/sw#1486] [Q&A] [answered]
Καλησπέρα κύριε @epidrome και συνάδελφοι έκλεισα ραντεβού για την Δευτέρα 3 Απρ. και θεωρώ ότι έως εκείνη την ημερομηνία θα έχω καταφέρει να ολοκληρώσω όλα τα παραδοτέα του μαθήματος. Θα υπάρχει κάποια βαθμολογική επίπτωση αν κάνω απευθείας τελική παρουσίαση χωρίς ενδιάμεση; (Έχω δήλωση ενδιάμεση ακριβώς για την περίπτωση την οποία θα επίπτωση για να αλλάξω και τον προγραμματισμό μου).

Σας ευχαριστώ όλους για τον χρόνο σας.

**Comment by epidrome** [comment, marked as answer]
Η «προκαταβολική» πριν την 10 εβδομάδα παρουσίαση της τελικής προόδου έχει μπόνους έναν βαθμό.
https://epidrome.github.io/teaching/progress/
